### PR TITLE
[docs] #159, #171, #179: Update config sections

### LINKED ISSUE #159: Add more details on the genesis block location and configuration
Documentation page to edit: "[Genesis Block](https://hyperledger.github.io/iroha-2-docs/guide/configure/genesis.html)"

### Details to add

* A `path` and `filename` for a file, containing the Genesis configuration given the current Docker configuration
* Iroha flags to point out the Genesis configuration on bare metal

### LINKED ISSUE #171: To clarify the details on the Genesis Block page

### LINKED ISSUE #179: Clarify setting up a domain for public blockchains on Client Configuration page
> How can I set up a domain for public blockchains

> I believe that this part is related to global DNS, so someone should rent a domain name from a [registrar](https://en.wikipedia.org/wiki/Domain_name_registrar), have a static IP per peer and assign an [A-record](https://www.cloudflare.com/learning/dns/dns-records/dns-a-record/) to said IP.
> Otherwise it may be about internal Iroha domains, but I doubt it. You may ask the person wrote this part after checking it with git blame. (Probably @outoftardis)

It wasn't written by me. 
@appetrosyan can you help with the paragraph below?

> If you are setting up an Iroha peer, you should set up a domain for public blockchains. Bare sockets are enough for a local private deployment.

_Originally posted by @outoftardis in https://github.com/hyperledger/iroha-2-docs/issues/170#issuecomment-1248020528_

### PR DESCRIPTION
**Pull request by outoftardis**
Signed-off-by: Ekaterina Mekhnetsova <mekkatya@gmail.com>

Closes #159 
Closes #171 
Closes #179

### CODE REVIEW

File: src/guide/advanced/isi.md [hunk 1]
```diff
@@ -13,15 +13,15 @@ available for each object.
...
-| [SetKeyValue/RemoveKeyValue](#setkeyvalueremovekeyvalue) | Update metadata of blockchain objects.            |
-| [Grant/Revoke](#grantrevoke)                             | Give or remove certain permissions from accounts. |
-| [Transfer](#transfer)                                    | Transfer assets between accounts.                 |
-| [ExecuteTrigger](#executetrigger)                        | Execute triggers.                                 |
-| [If, Pair, Sequence](#composite-instructions)            | Use to create composite instructions.             |
+| Instruction                                               | Descriptions                                      |
+| --------------------------------------------------------- | ------------------------------------------------- |
+| [Register/Unregister](#un-register)                       | Give an ID to a new entity on the blockchain.     |
+| [Mint/Burn](#mint-burn)                                   | Mint/burn assets, triggers, or permission tokens. |
+| [SetKeyValue/RemoveKeyValue](#setkeyvalue-removekeyvalue) | Update metadata of blockchain objects.            |
```
Comment (WRRicht3r) [inline src/guide/advanced/isi.md:20]:
```suggestion
| [SetKeyValue/RemoveKeyValue](#setkeyvalue-removekeyvalue) | Update blockchain object metadata.            |
```

File: src/guide/advanced/running-iroha-on-bare-metal.md [hunk 2]
```diff
@@ -13,6 +9,24 @@ The file-based approach is the easiest to get right. Using environment
 variables can offer a better user experience if done right, but is more
 error-prone, particularly for exotic systems (Windows).
 
+::: info
+
+For this chapter, we assume you have learned about
+[configuration and management](./../configure/intro.md) in Iroha 2. Here we
+offer you instructions for running Iroha on bare metal without going into
```
Comment (WRRicht3r) [inline src/guide/advanced/running-iroha-on-bare-metal.md:16]:
```suggestion
offer you instructions to run Iroha on bare metal without going into
```

File: src/guide/advanced/running-iroha-on-bare-metal.md [hunk 3]
```diff
@@ -13,6 +9,24 @@ The file-based approach is the easiest to get right. Using environment
...
+details about various configuration options available to you.
```
Comment (WRRicht3r) [inline src/guide/advanced/running-iroha-on-bare-metal.md:17]:
```suggestion
details about various configuration options available.
```

**Review by 6r1d — APPROVED** [review]
LGTM

File: src/guide/advanced/running-iroha-on-bare-metal.md [hunk 4]
```diff
@@ -13,6 +9,24 @@ The file-based approach is the easiest to get right. Using environment
...
+
+You can always check
+[sample configuration files](./../configure/sample-configuration.md) for
+`/configs/peer/genesis.json` and `/configs/peer/config.json`, or refer to
+[peer configuration options](./../configure/peer-configuration.md) for more
+details.
+
+You will find the complete list of options in
```
Comment (WRRicht3r) [inline src/guide/advanced/running-iroha-on-bare-metal.md:25]:
```suggestion
The complete list of options is available in the
```

File: src/guide/advanced/running-iroha-on-bare-metal.md [hunk 5]
```diff
@@ -434,12 +448,14 @@ Iroha in the real world.
...
+    array in your configuration file.
 
 4.  Determine the web socket that the other peers will use to connect to
-    you. Make sure that the port is open and use that Address in your
-    `config.json`.
+    you. Make sure that the port is open and use that address
+    ([`P2P_ADDR`](./../configure/peer-configuration.md#p2p_addr)) in your
+    `/configs/peer/config.json`.
 
 5.  After you finished editing the configuration file, deploy Iroha by
```
Comment (WRRicht3r) [inline src/guide/advanced/running-iroha-on-bare-metal.md:460]:
```suggestion
5.  After you have finished editing the configuration file, deploy Iroha by
```

File: src/guide/configure/client-configuration.md [hunk 6]
```diff
@@ -1,46 +1,116 @@
...
+::: info
+
+Learn how to [monitor Iroha performance](./../advanced/metrics.md) using
+prometheus.
+
+:::
+
+## Transaction Limits
+
+You can specify the transaction limits to which each transaction must
```
Comment (WRRicht3r) [inline src/guide/configure/client-configuration.md:91]:
```suggestion
You can specify the transaction limits that each transaction must
```

File: src/guide/configure/genesis.md [hunk 7]
```diff
@@ -50,6 +50,12 @@ automatically recommited upon container restart.
 
 ## Generation
 
+You can add various instructions to the genesis block, such as registering
+new accounts or assets, minting assets. You can also register permission
```
Comment (WRRicht3r) [inline src/guide/configure/genesis.md:54]:
```suggestion
new accounts or assets, as well as minting assets. You can also register permission
```

File: src/guide/configure/genesis.md [hunk 8]
```diff
@@ -50,6 +50,12 @@ automatically recommited upon container restart.
...
+tokens and roles as well as grant them to the registered accounts.
```
Comment (WRRicht3r) [inline src/guide/configure/genesis.md:55]:
```suggestion
tokens and roles, as well as grant them to the registered accounts.
```

File: src/guide/configure/peer-configuration.md [hunk 9]
```diff
@@ -119,6 +126,20 @@ You can do this via the configuration file (`ACCOUNT_PUBLIC_KEY`,
 Read more about [genesis blocks](./genesis.md) and
 [public key cryptography](./keys.md).
 
+Besides the public and private keys for genesis account, which are the
```
Comment (WRRicht3r) [inline src/guide/configure/peer-configuration.md:129]:
```suggestion
Aside from the public and private keys for the genesis account, which are
```

File: src/guide/configure/peer-configuration.md [hunk 10]
```diff
@@ -119,6 +126,20 @@ You can do this via the configuration file (`ACCOUNT_PUBLIC_KEY`,
...
+required configuration options, you can also fine-tune other genesis block
+configurations, such as:
+
+- `WAIT_FOR_PEERS_RETRY_COUNT_LIMIT`: the number of attempts to connect to
+  peers before genesis block is submitted
+- `WAIT_FOR_PEERS_RETRY_PERIOD_MS`: how long to wait before retrying
+  connecting to peers
```
Comment (WRRicht3r) [inline src/guide/configure/peer-configuration.md:136]:
```suggestion
  a connection to peers
```

File: src/guide/objects/assets.md [hunk 11]
```diff
@@ -31,8 +31,8 @@ related to [metadata](metadata.md).
 
 ## Instructions
 
-Assets can be [registered](../advanced/isi.md#unregister) and
-[minted/burned](../advanced/isi.md#mintburn).
+Assets can be [registered](../advanced/isi.md#un-register) and
+[minted/burned](../advanced/isi.md#mint-burn).
 
 Refer to one of the language-specific guide to walk you through the process
```
Comment (WRRicht3r) [inline src/guide/objects/assets.md:37]:
```suggestion
Refer to one of the language-specific guides to walk you through the process
```

**Review by WRRicht3r — CHANGES_REQUESTED** [review]

**Review by WRRicht3r — APPROVED** [review]

File: src/guide/configure/genesis.md [hunk 12]
```diff
@@ -11,7 +11,7 @@ empty, even if `configs/peer/genesis.json` is. Here's an example:
 
 The **genesis account** is specified in the
 [peer configuration](./peer-configuration.md#genesis) file,
-`configs/peer/config.json`. This is the account that will submit the
+`/configs/peer/config.json`. This is the account that will submit the
```
Comment (appetrosyan) [inline src/guide/configure/genesis.md:14]:
```suggestion
`configs/peer/config.json`. This is the account that will submit the
```
On Unix the first slash is the root directory. Please revert all instances.

**Review by appetrosyan — CHANGES_REQUESTED** [review]

**Review by appetrosyan — APPROVED** [review]

**Pull request merged**